components/search/SearchWidget.vue

```diff
@@ -64,7 +64,7 @@ function activate() {
 
 <template>
   <div ref="el" relative group>
-    <div bg-base border="~ base" h10 px-4 rounded-3 flex="~ row" items-center relative focus-within:box-shadow-outline gap-3>
+    <div bg-base border="~ base" h10 pl-4 rounded-3 flex="~ row" items-center relative focus-within:box-shadow-outline gap-3>
```

Comment:
don't use padding/margin left/right, check this please, we have some ltr languages: https://github.com/elk-zone/elk/blob/main/CONTRIBUTING.md#rtl-support

Reply:
Oh that is fantastic! Fixed.